azure_sdk/lib/profile_azure_stack/modules/links_profile_module.rb

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require 'azure_mgmt_links'
+
+module Azure::Profiles::Management::Profile_Azure_Stack
+  module Links
+    ResourceLinks = Azure::ARM::Links::Api_2016_09_01::ResourceLinks
+
+    module Models
+      ResourceLink = Azure::ARM::Links::Api_2016_09_01::Models::ResourceLink
+      ResourceLinkFilter = Azure::ARM::Links::Api_2016_09_01::Models::ResourceLinkFilter
+      ResourceLinkResult = Azure::ARM::Links::Api_2016_09_01::Models::ResourceLinkResult
+      ResourceLinkProperties = Azure::ARM::Links::Api_2016_09_01::Models::ResourceLinkProperties
+      Filter = Azure::ARM::Links::Api_2016_09_01::Models::Filter
+    end
+
+    #
+    # Links
+    #
+    class LinksClass
+      attr_accessor :resource_links, :configurable, :base_url, :options, :model_classes
```

Comment:
Curious: If this supposed to be `attr_accessor` or `attr_reader`. Is this open for read and write from client's usage perspective?

Reply:
Done

Reply:
As we synced up offline, Let's use @ instead of `self` to use `attr_reader`.

Reply:
Done